### DISCUSSION TITLE
British Standard Time zone

### DISCUSSION THREAD

**Discussion by gigingeorge** [apache/airflow#35756] [Q&A]
Can anyone help me, how to set BST zone on airflow. I already changed default_timezone in airflow.cfg to Europe/London. However, it's showing GMT time which is 1 hour behind BST and there is no direct BST option found

Thanks
Gigin George

**Comment by Taragolis** [comment]
Airflow internally use IANA timezone + UTC + maybe some short timezones which might use in pedulum.DateTime
So you could/should use IANA timezone, e.g. Europe/London

Europe/London use during approx of half year use British Summer Time, and half of year to UTC (GMT), and it only depend on provided date time. If we talk about current day then in the UK use UTC until March 31, 2024: https://www.zeitverschiebung.net/en/city/2643743

**Comment by gigingeorge** [comment]
Hi,
As per the time difference between UTC and BST in prenset day is 1 hour. How can I configure BST time in airflow

**Reply by Taragolis** [reply]
In present day Britain use UTC `¯\_(ツ)_/¯` this would be until [DST](https://en.wikipedia.org/wiki/Daylight_saving_time) in March 31, 2024

```python
>>> import pendulum
>>> dt1 = pendulum.datetime(2023, 11, 21, tz="Europe/London")
>>> dt1.utcoffset()
datetime.timedelta(0)

>>> dt2 = pendulum.datetime(2023, 10, 21, tz="Europe/London")
>>> dt2.utcoffset()
datetime.timedelta(seconds=3600)

>>> dt3 = pendulum.datetime(2024, 4, 1, tz="Europe/London")
>>> dt3.utcoffset()
datetime.timedelta(seconds=3600)
```

**Comment by bolkedebruin** [comment]
BST = British Summer Time. So if your IANA timezone is "Europe/London" and it is before 29 October 2023 and after 26 March 2023  it will be equal to "BST". British Winter Time = GMT. There is no British Standard Time.

https://en.wikipedia.org/wiki/Time_in_the_United_Kingdom
